docs/Routes.md

```diff
@@ -180,8 +179,32 @@ fastify.get('/', options, async function (request, reply) {
   reply.send(processed)
 })
 ```
+
+If the route is wrapping a callback-based API that will call
+`reply.send()` outside of the promise chain, it is possible to `await reply`:
+
+```js
+fastify.get('/', options, async function (request, reply) {
+  setImmediate(() => {
+    reply.send(processed)
+  })
+  await reply
+})
```

Comment:
Can you make this example a little bit more clear? Where is `processed` coming from?